documentation/src/main/asciidoc/topics/con_performance_clustered_cache_modes.adoc

```diff
@@ -0,0 +1,111 @@
+On a cluster of three nodes, two nodes can crash and you do not lose data from a replicated cache.
+
+In that same scenario, a distributed cache with two owners would lose data.
+To avoid data loss with distributed caches, you can increase the number of replicas across the cluster by configuring more owners for each entry.
+
+However you should also consider the performance penalty of rebalancing operations as well as the impact to capacity that occurs from node failure.
+When a node leaves the cluster, {brandname} replicates cache entries among the remaining members to restore the configured number of owners.
+This rebalancing operation is temporary but the increased cluster traffic does have a negative impact on performance.
+More nodes leave, greater performance degradation.
+Depending on different factors, such as cache configuration and resource allocation, the nodes left in the cluster might not have enough capacity to keep all data in memory when other nodes leave.
+
+.Cluster scaling
+
+{brandname} clusters scale horizontally as your workloads demand to more efficiently use compute resources like CPU and memory.
+To take the most advantage of this elasticity, you should consider how scaling the number of nodes up or down affects cache capacity.
+
+For replicated caches, each time a node joins the cluster it receives a complete copy of the data set.
+This means it takes more time for nodes to join but also imposes a limit on overall capacity because a replicated cache can never exceed the amount of memory available to the host.
+For example, if the size of your data set is 10Gb then each node must have at least 10Gb of available memory.
+
+On the other hand, adding more nodes increases capacity for a distributed cache as each member of the cluster stores only a subset of the data.
+To store 10Gb of data you can have eight nodes with 2.5Gb of available memory each if the number of owners is two.
+Each additional node that joins the cluster increases the capacity of the distributed cache by 2.5Gb.
+
+In this way, the capacity of a distributed cache is not bound by the amount of memory available to underlying hosts.
+
+.Synchronous or asynchronous
+
+{brandname} clusters can use synchronous or asynchronous communication to replicate entries between nodes.
+
+With synchronous communication, {brandname} notifies the originating node when the operation completes on the receiving node.
+
+{brandname} provides exceptions to applications when a replication fails.
+When a change to the cluster topology results in a failure, {brandname} retries the operation instead.
+This helps to keep your data consistent but adds latency to cluster traffic that reduces throughput for cache writes.
+
+With asynchronous communication, {brandname} does not provide any notification when it cannot replicate an entry.
+Applications always perceive asynchronous operations as successful, even if they fail.
+
```

Comment:
There is a queue where the missing backup replication is tracked. So the Powner will try to complete it.
Also it will have only the latest update in the queue (to save resources)